odoo/tools/config.py

```diff
@@ -124,6 +124,9 @@ def __init__(self, fname=None):
         group.add_option("--upgrade-path", dest="upgrade_path",
                          help="specify an additional upgrade path.",
                          action="callback", callback=self._check_upgrade_path, nargs=1, type="string")
+        group.add_option("--pre-update-scripts", dest="pre_update_scripts", my_default="",
```

Comment:
The CLI uses `update` for modules to upgrade which I think is quite confusing. Here since we already have  `--upgrade-path`, I prefer to use `--pre-upgrade-scripts`